geomesa-hbase/geomesa-hbase-datastore/src/test/scala/org/locationtech/geomesa/hbase/data/HBaseDensityFilterTest.scala

```diff
@@ -99,7 +99,8 @@ class HBaseDensityFilterTest extends HBaseTest with LazyLogging {
 
       val q = "(dtg between '2012-01-01T18:00:00.000Z' AND '2012-01-01T23:00:00.000Z') and BBOX(geom, -80, 33, -70, 40)"
       val density = getDensity(typeName, q, fs)
-      density.length must beLessThan(150)
+      density.length must beEqualTo(4)
```

Comment:
I'm hesitant to make this so precise - seems like an implementation detail we don't really care about testing, and at least with accumulo the number of iterators that get spun up isn't always the same